requirements.txt

```diff
@@ -3,3 +3,4 @@ django-model-utils
 swapper
 djangorestframework>=3.7,<3.8
 passlib
+djangorestframework-link-header-pagination
```

Comment:
bind the version as done in the `djangorestframework` line to avoid eventual future backward incompatible changes being installed automatically and hence breaking the package in production